app/src/androidTest/java/com/google/maps/android/compose/GoogleMapViewTests.kt

```diff
@@ -40,6 +40,7 @@ class GoogleMapViewTests {
     private lateinit var cameraPositionState: CameraPositionState
 
     private fun initMap(content: @Composable () -> Unit = {}) {
+        check(BuildConfig.MAPS_API_KEY != "YOUR_API_KEY") { "Maps API key not specified" }
```

Comment:
Maybe we should also check that MAPS_API_KEY is not blank, in case for some reason the default placeholder isn't read.

Reply:
Added!